docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #93.        while left_nn_distance &gt;= discord_score:
</pre><p>We should be able to avoid while loop here. Note that the condition is used again later in the body of while loop to break out of it. We can use a for-loop and consider a very generous upper bound for the number of iterations, and that is the total number of subsequences to the left of <code>query_idx</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>